frontend/src/core/comments/components/AddComment.js

```diff
@@ -2,8 +2,20 @@
 
 import * as React from 'react';
 import { Localized } from '@fluent/react';
+import Editor from 'draft-js-plugins-editor';
+import { 
+    EditorState, 
+    getDefaultKeyBinding, 
+    convertToRaw,
+    SelectionState,
+    Modifier, 
+} from 'draft-js';
+import draftToHtml from 'draftjs-to-html';
+import createLinkifyPlugin from 'draft-js-linkify-plugin';
 
 import './AddComment.css';
```

Comment:
I think we should also add the default `draft.js` CSS file:
```
import 'draft-js/dist/Draft.css';
```

That will greatly reduce our custom CSS rules, to only include ones that are absolutely needed to adapt the style for our use case. Here's the patch, which also prevents comment text in the editor from jumping by a pixel or two after text breaks into the 2nd line.

```diff
diff --git a/frontend/src/core/comments/components/AddComment.css b/frontend/src/core/comments/components/AddComment.css
index 2bdb48d3d..e59baed7a 100644
--- a/frontend/src/core/comments/components/AddComment.css
+++ b/frontend/src/core/comments/components/AddComment.css
@@ -11,7 +11,6 @@
     border-radius: 4px;
     color: #FFFFFF;
     cursor: text;
-    display: flex;
     flex-grow: 2;
     font-size: 11px;
     min-height: 26px;
@@ -22,18 +21,8 @@
     resize: none;
 }
 
-.DraftEditor-root {
-    height: 100%;
-    width: 100%;
-}
-
-.public-DraftEditorPlaceholder-root {
-    position: relative;
-    color: #aaa;
-}
-
 .public-DraftEditorPlaceholder-inner {
-    position: absolute;
+    color: #aaa;
 }
 
 .public-DraftEditor-content {
diff --git a/frontend/src/core/comments/components/AddComment.js b/frontend/src/core/comments/components/AddComment.js
index a8653a633..23c260790 100644
--- a/frontend/src/core/comments/components/AddComment.js
+++ b/frontend/src/core/comments/components/AddComment.js
@@ -14,6 +14,7 @@ import draftToHtml from 'draftjs-to-html';
 import createLinkifyPlugin from 'draft-js-linkify-plugin';
 
 import './AddComment.css';
+import 'draft-js/dist/Draft.css';
 import 'draft-js-linkify-plugin/lib/plugin.css';
 
```